app/presenters/bill-runs/two-part-tariff/review-licence.presenter.js

```diff
@@ -119,13 +119,24 @@ function _chargeReferenceDetails (reviewChargeVersion, chargePeriod) {
       chargeCategory: `Charge reference ${reviewChargeReference.chargeReference.chargeCategory.reference}`,
       chargeDescription: reviewChargeReference.chargeReference.chargeCategory.shortDescription,
       totalBillableReturns: _totalBillableReturns(reviewChargeReference),
+      chargeReferenceLink: _chargeReferenceLink(reviewChargeReference),
       chargeElements: _chargeElementDetails(reviewChargeReference, chargePeriod)
     })
   })
 
   return chargeReference
 }
 
+function _chargeReferenceLink (reviewChargeReference) {
+  const { charge: chargeFactor, aggregate } = reviewChargeReference.chargeReference.adjustments
+
+  if (chargeFactor || aggregate) {
+    return { linkName: 'Change details' }
+  }
+
+  return { linkName: 'View details' }
+}
+
```

Comment:
With no ticket so details coming by word of mouth this is my bad. But _just like_ volume, because we are going to amend these values the final design will require them being persisted to the review record alongside an amendable version as part of matching & allocating.

Using volume on the charge element as an example, there we have

- calculated: what the engine calculated
- allocated: defaults to calculated but this is what the user will change

We need the same for both aggregate and authorised volume on the charge reference. The one niggle for aggregate is when we look in the charge reference and find there is nothing we default the 'calculated' value to `1`.

So this obviously has implications on the logic. But also, when done the logic needs to be based on these values _not_ being equal to 1.